VoodooRMI/RMIBus.cpp

```diff
@@ -94,6 +94,14 @@ bool RMIBus::start(IOService *provider) {
     if (!transport->open(this))
         return false;
 
+    if (OSArray *config = transport->getConfig()) {
+        if (OSDictionary *dict = Configuration::mapArrayToDict(transport->getConfig())) {
+            updateConfiguration(dict);
+            dict->release();
+        }
+//        config->release();
```

Comment:
The retain count here is weird.

Reply:
I think since you only release once in getConfig but retain as well, you need to release here. You already know it's not NULL here so it's fine, I think.